sources/addr.json

```diff
@@ -3,8 +3,10 @@
 	"fullname": "addr",
 	"name": "addr",
 	"prefix": "addr",
-    	"key":"id",
+	"key": "id",
 	"url": "",
-	"license": "CC0",
-	"description": ""
-}
+	"license": "N/A",
+	"license_type": "CC0",
+	"license_text": "",
+	"description": "Property prefix."
```

Comment:
Should we be explicate that this is a Who's On First property prefix, or is that overkill?